operator/gitops/argocd/pipeline-service/kustomization.yaml

```diff
@@ -4,7 +4,6 @@ kind: Kustomization
 
 resources:
   - openshift-pipelines
-  - pipelines-as-code
```

Comment:
@bnallapeta As we don't need to install Pac as a separate resource. Do we need to delete its related manifest [files](https://github.com/openshift-pipelines/pipeline-service/tree/main/operator/gitops/argocd/pipeline-service/pipelines-as-code)

Reply:
yes, will do that later. That's why marked it as WIP. 
Wanted to get the tests triggered first.

Reply:
Done.